ignite/metrics/accuracy.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import division
+
+import torch
+
+from ignite.metrics.metric import Metric
+from ignite.exceptions import NotComputableError
+
+
+class Accuracy(Metric):
+    """
+    Calculates the accuracy.
+
+    - `update` must receive output of the form `(y_pred, y)`.
+    - `y_pred` must be in the following shape (batch_size, num_categories, ...)
```

Comment:
nit: may be we need to update this `(batch_size, num_categories, ...)` too

Reply:
resolved. let me know if that works.